Dispatcher/Repositories/ArticlesRepository.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  ArticlesRepository.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 30/04/2023.
+//
+
+import Foundation
+
+class ArticlesRepository {
+    
+    static let shared = ArticlesRepository()
+    let alamofireManager = AlamofireManager()
+    
+    private init() {}
+
+    func getArticlesFromApi(pageNumber: Int, completionHandler: @escaping (_ articles: [Article], _ totalPages: Int, _ errorMsg: String?) -> Void) {
+        let request = buildRequest(nil, pageNumber)
```

Comment:
When you don't provide names for the parameters that you send in it's much less readable and harder to understand. You actually have to go into the buildRequest function itself to understand that in this particular case you don't have search words and that's why you send nil into it.